modules/services/yabai/default.nix

```diff
@@ -62,41 +64,66 @@ in
       '';
     };
 
-    services.yabai.extraConfig = mkOption {
-      type = lines;
+    extraConfig = mkOption {
+      type = types.lines;
       default = "";
-      example = literalExpression ''
+      example = lib.literalExpression ''
         yabai -m rule --add app='System Preferences' manage=off
       '';
       description = "Extra arbitrary configuration to append to the configuration file";
     };
+
+    errorLogFile = mkOption {
+      type = types.path;
+      default = "/var/tmp/yabai.error.log";
+      example = "/Users/khaneliman/Library/Logs/yabai.log";
+      description = "Path to the yabai error log file";
+    };
+
+    outLogFile = mkOption {
+      type = types.path;
+      default = "/var/tmp/yabai.out.log";
+      example = "/Users/khaneliman/Library/Logs/yabai.log";
+      description = "Path to the yabai stdout log file";
+    };
```

Comment:
Is there a reason you picked `/var/tmp` over `/tmp` or `/var/log`? Upstream just puts them in `/tmp`, but `/var/log` (like you did for the scripting additions) also makes sense to me

Reply:
User agents dont have access to /var/log but the global daemon does.

Reply:
That makes sense. What about `/tmp` instead of `/var/tmp`? I don't really care either way, I've just never seen `/var/tmp` used before and didn't actually know that existed

Reply:
/tmp could be purged on reboots, /var/tmp wouldn't be. So depends on what you think the default behavior should be.

Reply:
Ack sorry something is up with my notifications so I missed this. My personal preference is `/tmp` because it's what upstream appears to use, I think it's more visible, and I think it's fine if these logs don't persist across a reboot.

That said, I don't have permissions to merge this, so we could leave it up to @Enzime